apps/src/templates/sectionProgress/standards/LessonStatusList.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import MultiCheckboxSelector from '../../MultiCheckboxSelector';
+import ProgressBox from '../ProgressBox';
+
+const styles = {
+  lessonListItem: {
+    display: 'flex',
+    flexDirection: 'row'
+  }
+};
+
+export class LessonStatusList extends Component {
+  static propTypes = {};
```

Comment:
is this here in anticipation of the propTypes we'll need when we connect to real data?

Reply:
Yup! I figure we will at somepoint want to get the lesson list and status of the lessons from redux/props so I just left it there.